README.md

```diff
@@ -315,3 +316,63 @@ Comp 40 gives you the tools to explore a large number of other topics:
 * Mark Sheldon gets very excited about some of the topics in Comp 40. Stop by his office to chat with him!
 * Norman Ramsey designed the class
 * Noah Mendelsohn is the [chairman of the internet](http://www.arcanedomain.com/computerwork.html) and is in general incredibly smart; he teaches COMP 40 on alternating semesters with Prof. Sheldon. Talk to him too!
+
+## COMP 15
+
+### Topics Covered:
+* Your first guide to getting a software engineering job (interview questions!)
+* Knowing the difference between the client side and the implementation side 
+* Introduction to C++ programs that utilize complex data structures
+* Utilizing Object Oriented Programming (OOP) concepts to write better code
```

Comment:
Maybe choose a more specific word than "better". How will your code be improved by the end of the class?